RedHat-UX/red-hat-design-system
# docs(elements): update images and summaries

Closes #1008

## What I did

1. Updated all the images and summaries for the `/elements` landing page according to the design [spec](https://docs.google.com/document/d/1udnh5wDxuzDV1NDY1QBg1tJMenTOSnntXU8x46t9Sec/edit)
2.  Modified the `example` shortcode to allow for absolute paths in its image src


## Testing Instructions

1. Review the linked spec and verify that it matches the deploy [preview](https://deploy-preview-1007--red-hat-design-system.netlify.app/elements)

## Notes to Reviewers
